ThroughputTest/README.md

```diff
@@ -0,0 +1,130 @@
+﻿# Azure Service Bus Throughput Performance Test
+
+This code sample illustrates the available throughput performance options for Azure Service Bus clients 
+and also allows for running experiments that dynamically adjust some tuning parameters. 
+
+> This tool is meant to be used with Azure Service Bus Premium and not with Azure Service Bus Standard.
+> Azure Service Bus Premium is designed to provide predictable performance, meaning that the results 
+> you measure with this tool are representative of the performance you can expect for your applications. 
+
+The sample assumes that you are generally familiar with the Service Bus .NET SDK and with how to set up 
+namespaces and entities within those namespaces. 
+
+The sample can either send and receive messages from a single instance, or just act as either sender or receiver,
+allowing simulation of different scenarios. 
+
+The sending side supports sending messages singly or in batches, and it supports pipelining of send operations 
+whereby up to a certain number of messages are kept in flight and their acknowledgement is handled asynchronously. 
+You can start one or multiple concurrent senders, and each sender can be throttled by imposing a pause between
+individual send operations.
+
+The receive side supports the "ReceiveAndDelete" and "PeekLock" receive modes, one or multiple concurrent 
+receive loops, single or batch receive operations, and single or batch completion. You can simulate having
+multiple concurrent handlers on a receiver and you can also impose a delay to simulate work.
```

Comment:
>simulate work. [](start = 78, length = 14)

Would add a comment to mention that the standard expected mode is PeekLock. People should know but maybe don't.